src/pages/test-results-search/test-results-search.ts

```diff
@@ -87,6 +92,12 @@ export class TestResultsSearchPage extends BasePageComponent {
             this.store$.dispatch(new SaveLog(log));
             this.searchResults = [];
             this.showSearchSpinner = false;
+
+            if (err) {
+              this.showError(err);
+              return of(this.hasSearched = false);
```

Comment:
Do you mean to do assignment yielding a value here? Or is it meant to be a comparison?

Reply:
Yes, it's mean't to be an assignment. Moved to outside of return statement for clarity.